packages/melody-compiler/__tests__/__fixtures__/success/empty_attribute.template

```diff
@@ -0,0 +1,3 @@
+<div itemscope itemtype="http://schema.org/Hotel">
```

Comment:
I'd really like to see end to end tests for this to see which attributes will work that way and which won't.